### PR TITLE
add about page

### PR DESCRIPTION
**Pull request by ryuash**
## Checklist
- [ ] Ran Linting
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:3000",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (ryuash) [inline .vscode/launch.json:15]: Why is this here

File: package.json [hunk 2]
```diff
@@ -26,11 +26,15 @@
     "next": "9.5.3",
     "next-i18next": "^6.0.2",
     "ramda": "^0.27.1",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-i18next": "^11.7.2"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-i18next": "^11.7.2",
+    "subscriptions-transport-ws": "^0.9.18"
```
Comment (ryuash) [inline package.json:32]: ????

File: package.json [hunk 3]
```diff
@@ -26,11 +26,15 @@
...
   },
   "devDependencies": {
+    "@babel/core": "^7.11.6",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
```
Comment (ryuash) [inline package.json:37]: ??

File: package.json [hunk 4]
```diff
@@ -52,9 +57,13 @@
     "jest": "^26.4.2",
     "jest-watch-typeahead": "^0.6.1",
     "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
```
Comment (ryuash) [inline package.json:60]: ??

File: src/components/layout/index.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ export const Layout = (props: any) => {
   return (
     <div>
       {children}
-      <Footer />
+      <Footer/>
```
Comment (ryuash) [inline src/components/layout/index.tsx:10]: uh Do you have a linter?

File: src/pages/index.tsx [hunk 6]
```diff
@@ -1,8 +1,9 @@
 import Home from '@screens/home';
+import About from '@screens/about';
```
Comment (ryuash) [inline src/pages/index.tsx:2]: remove

File: src/screens/about/hooks.tsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  const {
+    loading, error, data,
+  } = useQuery(queryExample);
+
+  return {
+    loading,
+    error,
+    data,
+  };
+};
```
Comment (ryuash) [inline src/screens/about/hooks.tsx:14]: This should not be in about

File: src/screens/about/index.tsx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+          className={classes.textMobile}
+
+        ><span>{t('paragraph')}</span>
+        </div>
+      </div>
+      <Example />
+
+    </Layout>
+  );
+};
```
Comment (ryuash) [inline src/screens/about/index.tsx:73]: There is a lot of useless repetition that isn't needed at all (i'll explain more in basecamp)

File: src/screens/about/styles.tsx [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+    listItem: {
+      whiteSpace: 'nowrap',
+      width: 'auto',
+      padding: '0',
+      fontSize: theme.typography.body3.fontSize,
+      '& span': {
+        margin: '0 1rem',
+      },
+    },
+    playStore: {
```
Comment (ryuash) [inline src/screens/about/styles.tsx:59]: i'm pretty sure this is ripped directly off of the footer styles. You don't need this much

File: src/screens/about/utils.tsx [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      {
+        image: 'images/google-play-badge.svg',
+        url: '#',
+        className: classes.googleBadge,
+        alt: 'google store',
+      },
+    ]
+  );
+};
```
Comment (ryuash) [inline src/screens/about/utils.tsx:68]: delete this. Not needed here

File: src/styles/theme.ts [hunk 11]
```diff
@@ -21,6 +21,25 @@ export const common = {
     custom: {
       white: 'rgba(255, 255, 255, 1)',
     },
+    about: {
+      main: 'rgba(25, 55, 81, 1)',
+    },
+  },
```
Comment (ryuash) [inline src/styles/theme.ts:27]: nope this isn't how the styles work. (ill explain in basecamp)

File: src/styles/theme.ts [hunk 12]
```diff
@@ -21,6 +21,25 @@ export const common = {
...
+    desktop: 1024,
+  },
+  typography: {
+    body3: {
+      fontSize: 12,
+    },
+  },
+  margin: {
+    paddingbig: '20px',
+    paddingmidium: '10px',
```
Comment (ryuash) [inline src/styles/theme.ts:42]: Our designer just uploaded the palette so let's check on this

File: .vscode/launch.json (hunk 1 above)
Comment (YuhsinLiao1995) [reply .vscode/launch.json:15]: umm becuz i ran it with debug mode so it was created automatically, i'll delete it.

File: package.json (hunk 2 above)
Comment (ryuash) [reply package.json:32]: really just the `subscriptions-transport-ws` what's this for?

File: src/screens/about/styles.tsx (hunk 9 above)
Comment (YuhsinLiao1995) [reply src/screens/about/styles.tsx:59]: i'll clean it up.

File: package.json (hunk 4 above)
Comment (YuhsinLiao1995) [reply package.json:60]: I'll uninstall it

Comment (ryuash) [reply package.json:60]: Can you explain why you installed it though

File: src/styles/theme.ts (hunk 12 above)
Comment (YuhsinLiao1995) [reply src/styles/theme.ts:42]: I'm not sure how to revise this part? should i follow this doc? 
截圖 2020-10-07 下午2 43 21

File: src/styles/theme.ts (hunk 11 above)
Comment (YuhsinLiao1995) [reply src/styles/theme.ts:27]: revised it!

File: src/screens/about/utils.tsx (hunk 10 above)
Comment (YuhsinLiao1995) [reply src/screens/about/utils.tsx:68]: deleted it.
